CONTRIBUTING.md

```diff
@@ -153,10 +153,12 @@ If you are a VS Code user, you may want to add the following to your `.vscode/se
 Every fix, improvement, feature, or breaking change should be made in a
 pull-request that includes an update to the `CHANGELOG_PENDING.md` file.
 
-Changelog entries should be formatted as follows:
+Changelog entries should answer the question: "what is important about this
+change for users to know?" and should not simply be a reiteration of the title
+of the associated PR. Changelog entries should be formatted as follows:
```

Comment:
I'm wondering if you could provide some examples? In general, what do we think it's important for our users to understand beyond what the change was? I guess I'm not clear on how this is that distinct from the title of the pull request.

Reply:
Would links to some resources like https://keepachangelog.com/en/1.0.0/ perhaps help? It seems to me like what I'm asking for here is pretty standard, but perhaps it's not as well-understood as I thought?